mlflow/environment_variables.py

```diff
@@ -762,8 +762,9 @@ def get(self):
 )
 
 
-#: Specified the ID of the LoggedModel to link traces to.
+#: Default active LoggedModel ID.
 #: This should only by used by MLflow internally or in standalone environments such
-#: as Databricks serving.
+#: as Databricks serving, users should always use `set_active_model` to set the
+#: active LoggedModel, and should not set this environment variable directly.
 #: (default: ``None``)
 MLFLOW_ACTIVE_MODEL_ID = _EnvironmentVariable("MLFLOW_ACTIVE_MODEL_ID", str, None)
```

Comment:
```suggestion
_MLFLOW_ACTIVE_MODEL_ID = _EnvironmentVariable("MLFLOW_ACTIVE_MODEL_ID", str, None)
```

nit: since this is an internal env variable now